routes/movies.routes.js

```diff
@@ -0,0 +1,105 @@
+const Movie = require("../models/Movie.model");
+const Celebrity = require("../models/Celebrity.model");
+const router = require("express").Router();
+
+//READ movies
+router.get("/movies", (req, res, next) => {
+    Movie.find()
+        .then((moviesArr) => {
+            res.render("movies/movies", {
+                movies: moviesArr
+            })
+        })
+        .catch((err) => {
+            console.log("Oops, there was an error!", err)
+        })
+
+})
+
+//CREATING new movies
+
+router.get("/movies/create", (req, res, next) => {
+    res.render("movies/new-movie")
+})
+
+router.post("/movies/create", (req, res, next) => {
+
+    const newMovie = {
+        title: req.body.title,
+        genre: req.body.genre,
+        plot: req.body.plot,
+        cast: req.body.cast,
+    }
+
+    Movie.create(newMovie)
+        .then((movieFromDB) => {
+            res.redirect("/movies")
+        })
+        .catch((err) => {
+            console.log("Oops, there was an error!", err)
+            res.render("movies/new-movie")
+        })
+})
+
+//SHOW movie details
+
+router.get("/movies/:id", (req, res, next) => {
+    const id = req.params.ObjectId
```

Comment:
req.params  reads the variable name that you have in the route , which in your case is `id`

```
router.get("/movies/:id", (req, res, next) => {
        const id = req.params.id
```